tests/integration_tests/test_mlflow.py

```diff
@@ -96,6 +96,37 @@ def test_study_name(tmpdir: py.path.local) -> None:
     assert first_run_dict["data"]["tags"]["my_user_attr"] == "my_user_attr_value"
 
 
+@pytest.mark.parametrize("name,expected", [(None, "Default"), ("foo", "foo")])
+def test_use_existing_or_default_experiment(
+    tmpdir: py.path.local, name: Optional[str], expected: str
+) -> None:
+
+    if name is not None:
+        tracking_file_name = "file:{}".format(tmpdir)
```

Comment:
[Note] This is out-of-scope of this PR, but the `tracking_file_name` is actually a URI, not a file name. I think you follow the naming convention of existing test cases, so I accept the change in this PR. Maybe we work on it in a follow-up PR.

Reply:
I [commented](https://github.com/optuna/optuna/pull/2912#issuecomment-915462924) about the need to refactor some of the tests and I'm aiming to tackle it some time soon, so will have this comment in mind!